google/cloud/storage/examples/storage_async_samples.cc

```diff
@@ -394,6 +394,7 @@ void ResumeRewrite(google::cloud::storage_experimental::AsyncClient& client,
                 << progress.object_size << " bytes rewritten\n";
       if (progress.metadata) co_return *std::move(progress.metadata);
     }
+    throw std::runtime_error("rewrite interrupted without completing");
```

Comment:
comment: above you say "failed" instead of "interrupted"

Reply:
Fixed.